tests/integration/utils.ts

```diff
@@ -110,7 +110,7 @@ export const devBob = Crypto.makeKeypairFromUri('//Bob')
 export const devCharlie = Crypto.makeKeypairFromUri('//Charlie')
 
 export function addressFromRandom(): KiltAddress {
-  return encodeAddress(randomAsU8a())
+  return encodeAddress(randomAsU8a()) as KiltAddress
```

Comment:
```suggestion
  return encodeAddress(randomAsU8a(), 38)
```
That was a bug that the fix to the `encodeAddress` typings has revealed

Reply:
[08046d2](https://github.com/KILTprotocol/sdk-js/pull/890/commits/08046d292c3a7c2358c0bdd1a0a000969e76ea22)

Reply:
[08046d2](https://github.com/KILTprotocol/sdk-js/pull/890/commits/08046d292c3a7c2358c0bdd1a0a000969e76ea22)